src/semgrep_agent/semgrep_app.py

```diff
@@ -74,7 +75,7 @@ def report_start(self, meta: GitMeta) -> Optional[str]:
                 ignore_patterns=glom(body, T["scan"]["meta"].get("ignored_files", [])),
             )
             debug_echo(f"=== Our scan object is: {self.scan!r}")
-            return str(glom(body, T["policy"])) if "policy" in body else None
+            return cast(Optional[str], glom(body, T["policy"], default=None))
```

Comment:
hahaha it's exactly the same amount of characters in the end

but we did it :D